lib/screens_web/controllers/v2/screen_controller.ex

```diff
@@ -35,10 +35,11 @@ defmodule ScreensWeb.V2.ScreenController do
     put_layout(conn, {ScreensWeb.V2.LayoutView, "app.html"})
   end
 
-  def index(conn, %{"id" => screen_id}) do
+  def index(%{params: params} = conn, %{"id" => screen_id}) do
```

Comment:
Same here, let's bind the second arg to `params`.
```suggestion
  def index(conn, %{"id" => screen_id} = params) do
```